nodes/mainnet.md

```diff
@@ -162,6 +164,7 @@ to participate in Mainnet Beta:
 - `celestia.grpc.archives.validao.xyz:9090`
 - `gprc-archive.celestia.bitszn.com`
 - `celestia-grpc.f5nodes.com`
+- `celestia-grpc.chainode.tech:443`
```

Comment:
```suggestion
- `celestia-grpc.chainode.tech:443`

#### Websocket endpoints

- `wss://celestia-ws.chainode.tech:33373/websocket`
```

Reply:
Sure, we can do this. Will commit the suggestion. Also, ideally the endpoints should have the exact protocol they are using http or https / ws or wss. Or there is the assumption that all endpoints are TLS encrypted.